data/policies/networking_rules.cas

```diff
@@ -0,0 +1,7 @@
+resource my_port {
+	portcon("tcp", 1234);
```

Comment:
I would add an error case test make sure we are erroring correctly on tcp/udp.  Also do you think it's worthwhile to check that the port is in the valid port range?